pages/form-builder/contactus.tsx

```diff
@@ -0,0 +1,281 @@
+  };
+
+  const validationSchema = Yup.object().shape({
+    name: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .max(50, t("signUpRegistration.fields.name.error.maxLength")),
+    email: Yup.string()
+      .required(t("input-validation.required", { ns: "common" }))
+      .email(t("input-validation.email", { ns: "common" })),
+    request: Yup.string().required(t("input-validation.required", { ns: "common" })),
+    description: Yup.string().required(t("input-validation.required", { ns: "common" })),
+  });
+
+  return (
+    <div aria-live="polite">
+      {!isSuccessScreen && (
+        <Formik
+          initialValues={{ name: "", email: "", request: "", description: "" }}
+          onSubmit={async ({ name, email, request, description }) => {
+            setSubmitting(true);
+            try {
+              const response = await handleRequest(name, email, request, description);
+              setSubmitting(false);
+              if (response?.status !== 200) {
+                throw new Error(t("submissionError"));
+              }
+              setErrorState({ message: "" });
+              setIsSuccessScreen(true);
+            } catch (err) {
+              logMessage.error(err);
+              setSubmitting(false);
+              setErrorState({ message: t("submissionError") });
+            }
+          }}
+          validateOnChange={false}
+          validateOnBlur={false}
+          validationSchema={validationSchema}
+        >
+          {({ handleSubmit, errors }) => (
+            <>
+              {errorState.message && (
+                <Alert
+                  type="error"
+                  validation={true}
+                  tabIndex={0}
+                  id="requestSubmissionError"
+                  heading={t("input-validation.heading", { ns: "common" })}
+                >
+                  {errorState.message}
+                </Alert>
+              )}
+
+              {Object.keys(errors).length > 0 && (
+                <Alert
+                  type="error"
+                  validation={true}
+                  tabIndex={0}
+                  id="validationErrors"
+                  heading={t("input-validation.heading", { ns: "common" })}
+                >
+                  <ol className="gc-ordered-list">
+                    {Object.entries(errors).map(([fieldKey, fieldValue]) => {
+                      return (
+                        <ErrorListItem
+                          key={`error-${fieldKey}`}
+                          errorKey={fieldKey}
+                          value={fieldValue}
+                        />
+                      );
+                    })}
+                  </ol>
+                </Alert>
+              )}
+
+              <h1>{t("contactus.title")}</h1>
+              <p className="mb-6 mt-[-2rem]">
+                {t("contactus.paragraph1")}&nbsp;
+                <Link href={`/${i18n.language}/form-builder/support`}>
```

Comment:
No need to actually include the language here sine you're using the Link component. Next will already have the context of the language the user has selected and will direct them to the appropriate language regardless